doc/getting-started/gsg_quick_setup.sh

```diff
@@ -0,0 +1,207 @@
+#!/bin/bash
+# Copyright (C) 2018 Intel Corporation.
+# SPDX-License-Identifier: BSD-3-Clause
+# This script is aim to setup or upgrade sos & uos automatically.
+
+# switch for download uos function
+skip_download_uos=0
+# turn on if you need to setup uos
+upgrade_uos=0
+
+function print_help()
+{
+echo "Usage:"
+echo "Launch this script as: ./flash.sh -t 28100"
+echo "-t to specify clear linux version for upgrading"
```

Comment:
It would improve readability to add a `tab` at the beginning of each parameter line, e.g. 
```suggestion
echo -e "\t-t to specify clear linux version for upgrading"
```